idr/externalDBs.py

```diff
@@ -10,26 +10,31 @@
 from IPython.display import Image
 
 
-def genes_of_interest_from_string(gene_names,
-                                  no_of_interacting_partners, taxonomyid):
-    url = 'https://string-db.org/api/psi-mi-tab/interactionsList?identifiers='
+def genes_of_interest_from_string(gene_names, no_of_interacting_partners, taxonomyid):
+    url = "https://string-db.org/api/psi-mi-tab/interaction_partners?identifiers="
     for g in gene_names:
-        url = url + g + '%0d'
-    url = (url + '&limit=' + str(no_of_interacting_partners) +
-           '&network_flavor=evidence&species=' + taxonomyid)
+        url = url + g + "%0d"
+    url = (
+        url
+        + "&limit="
+        + str(no_of_interacting_partners)
+        + "&network_flavor=evidence&species="
+        + taxonomyid
+    )
     res = requests.get(url)
-    df = read_csv(StringIO(res.text), sep='\t', header=None)
+    pandas.set_option("display.max_columns", 500)
```

Comment:
```suggestion
```
I don't think we should be overriding global presentation settings inside a library, instead it should be done in the notebook that requires it.

Reply:
I think I tested it there first when I was investigating the problem and forgot to remove it.